server/db/models/nyt.js

```diff
@@ -0,0 +1,10 @@
+const Sequelize = require('sequelize');
+const db = require('../builder');
+
+const Nyt = db.define('nyt', {
```

Comment:
@AnnaC91 @ErikaNewland what do you think, think this should be `Article` or `Headline`  and we can put a type column as `New York Times` I think a model should reference what is actually in it vs. the type

Reply:
Ok, @AnnaC91 let's change this Model to Headlines add a new 'type' column that is a sequelize.string